Trusted setup failing

<!--- Hello,-->

<!--- please note that this is an issue tracker reserved for bug reports and feature requests. -->

# Description

When attempting to run a trusted setup for a single folder, the script fails (see error below). I haven't tested whether the script also fails if a 'full' trusted setup of all files is attempted.

```
$ ./nightfall-generate-trusted-setup
~/git/nightfall-demo/nightfall/zkp/code/gm17 ~/git/nightfall-demo/nightfall
~/git/nightfall-demo/nightfall
*** List of trusted setups ***
1 ft-batch-transfer
2 ft-burn
3 ft-mint
4 ft-transfer
5 nft-burn
6 nft-mint
7 nft-transfer
Completed trusted setups are: ft-batch-transfer ft-burn ft-mint ft-transfer nft-burn nft-mint nft-transfer 
Do you want to generate all trusted setups? (y/n) : n

Type the number of each trusted setup seperated by a comma
(E.g. type 1,2 to run the trusted setups for ft-batch-transfer & ft-burn) : 4

> nightfall@1.0.0 setup /Users/em137vl/git/nightfall-demo/nightfall
> docker-compose run zkp npm run setup -- "-i" "gm17/ft-transfer"

Creating network "nightfall_network" with the default driver
Creating volume "nightfall_mongo-nightfall-volume" with default driver
Creating volume "nightfall_mongo-merkle-tree-volume" with default driver
Creating volume "nightfall_zkp-code-volume" with local driver
Creating nightfall_ganache_1 ... done

> zkp@1.0.0 setup /app
> NODE_ENV=setup npx babel-node code/index.js "-i" "gm17/ft-transfer"

[BABEL] Note: The code generator has deoptimised the styling of /app/node_modules/lodash/lodash.js as it exceeds the max of 500KB.
/app/node_modules/@babel/core/lib/transformation/normalize-file.js:209
    throw err;
    ^

SyntaxError: /app/node_modules/tar/tar.js: Legacy octal literals are not allowed in strict mode (108:10)

  106 | // values for the mode field
  107 | var modes =
> 108 |   { suid: 04000 // set uid on extraction
      |           ^
  109 |   , sgid: 02000 // set gid on extraction
  110 |   , svtx: 01000 // set restricted deletion flag on dirs on extraction
  111 |   , uread:  0400
    at Parser.raise (/app/node_modules/@babel/parser/lib/index.js:6344:17)
    at Parser.readNumber (/app/node_modules/@babel/parser/lib/index.js:7194:14)
    at Parser.getTokenFromCode (/app/node_modules/@babel/parser/lib/index.js:6966:14)
    at Parser.nextToken (/app/node_modules/@babel/parser/lib/index.js:6542:12)
    at Parser.next (/app/node_modules/@babel/parser/lib/index.js:6482:10)
    at Parser.eat (/app/node_modules/@babel/parser/lib/index.js:6487:12)
    at Parser.parseObjectProperty (/app/node_modules/@babel/parser/lib/index.js:9264:14)
    at Parser.parseObjPropValue (/app/node_modules/@babel/parser/lib/index.js:9290:101)
    at Parser.parseObjectMember (/app/node_modules/@babel/parser/lib/index.js:9214:10)
    at Parser.parseObj (/app/node_modules/@babel/parser/lib/index.js:9138:25)
    at Parser.parseExprAtom (/app/node_modules/@babel/parser/lib/index.js:8774:21)
    at Parser.parseExprSubscripts (/app/node_modules/@babel/parser/lib/index.js:8413:23)
    at Parser.parseMaybeUnary (/app/node_modules/@babel/parser/lib/index.js:8393:21)
    at Parser.parseExprOps (/app/node_modules/@babel/parser/lib/index.js:8280:23)
    at Parser.parseMaybeConditional (/app/node_modules/@babel/parser/lib/index.js:8253:23)
    at Parser.parseMaybeAssign (/app/node_modules/@babel/parser/lib/index.js:8200:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zkp@1.0.0 setup: `NODE_ENV=setup npx babel-node code/index.js "-i" "gm17/ft-transfer"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zkp@1.0.0 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-01-08T09_57_38_460Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! nightfall@1.0.0 setup: `docker-compose run zkp npm run setup -- "-i" "gm17/ft-transfer"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the nightfall@1.0.0 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/em137vl/.npm/_logs/2020-01-08T09_57_39_285Z-debug.log
```

--- a/nightfall-generate-trusted-setup
+++ b/nightfall-generate-trusted-setup
@@ -40,7 +40,7 @@ generateSetups(){
   selectedSetupsIndexes=$(echo $selectedSetupsIndexes | tr "," "\n")
   for i in $selectedSetupsIndexes
   do
-    npm run setup -- -i gm17/${setups[$i-1]}
+    npm run setup -- -f gm17/${setups[$i-1]}
   done
 }
 

--- a/zkp/code/index.js
+++ b/zkp/code/index.js
@@ -18,16 +18,16 @@ const getDirectories = source =>
     .filter(isDirectory);
 
 /**
- * Trusted setup for Nightfall. Either compiles all directories in /code/gm17, or a single directory using the -i flag.
+ * Trusted setup for Nightfall. Either compiles all directories in /code/gm17, or a single directory using the -f flag.
  * Calls zokrates' compile, setup, and export-verifier on all (or a specified) directories in `/zkp/code/gm17`.
  */
 async function main() {
-  // -i being the name of the .code file (i.e., 'ft-mint')
-  const { i } = argv;
+  // -f being the name of the .code file (i.e., 'ft-mint')
+  const { f } = argv;
 
-  if (!i) {
+  if (!f) {
     console.log(
-      "The '-i' option has not been specified.\nThat's OK, we can go ahead and loop through every .code file.\nHOWEVER, if you wanted to choose just one file, cancel this process, and instead use option -i (see the README-trusted-setup)",
+      "The '-f' option has not been specified.\nThat's OK, we can go ahead and loop through every .code file.\nHOWEVER, if you wanted to choose just one file, cancel this process, and instead use option -f (see the README-trusted-setup)",
     );
     console.log('Be warned, this could take up to an hour!');
 
@@ -55,7 +55,7 @@ async function main() {
       throw new Error(`Trusted setup failed: ${err}`);
     }
   } else {
-    await generateZokratesFiles(`${process.cwd()}/code/${i}`);
+    await generateZokratesFiles(`${process.cwd()}/code/${f}`);
   }
 }
 

--- a/zkp/package.json
+++ b/zkp/package.json
@@ -5,8 +5,8 @@
   "repository": "https://github.com/EYBlockchain/nightfall",
   "main": "index.js",
   "scripts": {
-    "setup": "NODE_ENV=setup npx babel-node code/index.js",
-    "setup-all": "yes | NODE_ENV=setup npx babel-node code/index.js",
+    "setup": "NODE_ENV=setup babel-node code/index.js",
+    "setup-all": "yes | NODE_ENV=setup babel-node code/index.js",
     "start": "nodemon --ignore src/vkIds.json --ignore stats-config/ --ignore code/ --exec babel-node ./src/index.js",
     "test": "jest --verbose --runInBand --forceExit ./__tests__/*.test.js"
   },

